UI/src/utils/backend-utils.js

```diff
@@ -610,7 +614,21 @@
     };
 
     this.getGeocode = function(address) {
-      return $.post("vkm/geocode", { address: address }).then(function(x) { return JSON.parse(x); });
+      var addressNormalized = address.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
+      var parsedAddress = addressNormalized.split(/^(\s*\w.*)(\s)(\s*\d+\s*),(\s*\w.*)/)
+          .filter( function(elem) {  return !_.isEmpty(elem.trim()); })
+          .map(function(elem) { return elem.trim(); });
+
+      return this.getMunicipalityIdByName(parsedAddress[2]).then(
+          function(kuntakoodi) {
```

Comment:
change the parameter name to municipalityCode or something else in english

Reply:
Since the getMunicipalityIdByName returns the municipalityInfo(id, ely and name) maybe itsbetter to call it municipalityInfo in my opinion

Reply:
Applied last comment